<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RR3O2CG" failures="1" tests="1" name="Test(failed)" time="31.532" errors="0" timestamp="2025-08-10T21:57:36 IST">
  <testcase classname="testCases.TC_07_CheckOutProductTest" name="verify_checkoutProcess" time="16.188">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false],
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false],
expected [true] but found [false]
at testCases.TC_07_CheckOutProductTest.verify_checkoutProcess(TC_07_CheckOutProductTest.java:115)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_checkoutProcess -->
</testsuite> <!-- Test(failed) -->
